src/components/legend/attributes.js

```diff
@@ -90,6 +90,33 @@ module.exports = {
         ].join(' ')
     },
 
+    itemclick: {
+        valType: 'enumerated',
+        values: ['toggle', 'focus', false],
+        dflt: 'toggle',
+        role: 'info',
+        editType: 'legend',
+        description: [
+            'Determines the behavior on legend item click.',
+            '*toggle* toggles the visible of the item clicked on the graph.',
+            '*focus* makes the clicked item the sole visible item on the graph.',
+            '*false* disable legend item click interactions.'
+        ].join(' ')
+    },
+    itemdoubleclick: {
```

Comment:
- `clickmode` and `doublemode` could work too if someone has a strong opinion about this
- `'isolate'` in place of `'focus'` could work as well.

Reply:
@nicolaskruchten do you have any preference for this one?

Reply:
`toggleothers` ?

Reply:
See https://github.com/plotly/plotly.js/pull/3862#issuecomment-492843307 below which is related to this thread.

Reply:
Thanks for your https://github.com/plotly/plotly.js/pull/3862#issuecomment-492843307 - I'm also starting to think `'toggleothers'` might be best to describe the current double-click behaviour. 

Does anyone _object_?

Reply:
Done in https://github.com/plotly/plotly.js/pull/3862/commits/bde19eea4206211b34cef30eb39734f051992a9a